models/article_model.js

```diff
@@ -1,11 +1,34 @@
 const db = require("../db/connection")
 
+function listArticles() {
+  const queryStr = `
+    SELECT
+      articles.author,
+      articles.title,
+      articles.article_id,
+      articles.topic,
+      articles.created_at,
+      articles.votes,
+      articles.article_img_url,
+      COUNT(comments.comment_id) AS comment_count
+      FROM articles
+      LEFT JOIN comments ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at DESC;
+    `;
+
+  return db.query(queryStr).then((articlesData) => {
+    console.log(articlesData.rows);
+    return articlesData.rows;
+  });
+}
```

Comment:
This is a great query you have constructed in your model! 🏆 
It would be good to remove the console log once you finish developing 🧹 and just a note that this will give you the `comment_count` as a `string`, you can get it as a `number` but it requires an extra bit of SQL syntax to be added to the query 👀